lib/relation-definition.js

```diff
@@ -2391,9 +2391,16 @@ RelationDefinition.embedsMany = function embedsMany(modelFrom, modelTo, params)
     embed: true,
   });
 
-  modelFrom.dataSource.defineProperty(modelFrom.modelName, propertyName, {
-    type: [modelTo], default: function() { return []; },
-  });
+  var opts = Object.assign(
+    {},
```

Comment:
Since the old code doesn't get `opts` from `params.options.property`, it's good to just use `{}` here 👌